Library/Homebrew/utils/user.rb

```diff
@@ -0,0 +1,12 @@
+require "system_command"
+require "etc"
+
+USER = Etc.getlogin
+
+def USER.gui?
+  out, _, status = system_command "who"
+  return false unless status.success?
+  out.split("\n")
+     .map { |line| line.split(/\s+/, 3) }
```

Comment:
Why 3?

Reply:
Name, type, and time (which we don't need).

Reply:
Could that get a comment or variable/constant name?